### ISSUE TITLE
Unable to redirect bypass login when tapped on Button

### ISSUE THREAD

**Issue by Vikassingamsetty** [openid/AppAuth-iOS#670]
When User taps on login button. I am calling 
`OIDAuthorizationService.discoverConfiguration(forIssuer: issuer) { [self] configuration, error in
            print("Start : \(configuration)")
            guard let config = configuration else {
                print("Error retrieving discovery config document: \(error?.localizedDescription ?? "Unknown error")")
                return
            }
            
            // perform the auth request...
            print("Start : \(config)")
            
            if let client = clientID {
                self.doAuthWithAutoCodeExchange(configuration: config, clientID: client, clientSecret: nil, params:params, vc:vc)
            }
            
        }` method and passing the URL, we get configuration.
Once configuration is obtained the I will be calling the below method with params `{idpHint: "INFY-AD-OIDC"}` value as dictionary 
`OIDAuthorizationRequest(configuration: configuration,
                                              clientId: clientID,
                                              clientSecret: clientSecret,
                                              scopes: [OIDScopeOpenID],
                                              redirectURL: redirectURI,
                                              responseType: OIDResponseTypeCode,
                                              additionalParameters: params)`

Here I will be getting id Token, Refresh token, Access token.
Basically with the params It has to redirect to MFA(Microsoft Authentication page) instead it is staying on the same page.
Please do suggest me how can I bypass navigation to MFA.
